### ISSUE TITLE
The controls such as "Repository" and "License" are not announced with proper role for screen reader users. 

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode#108302]
**“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”**

GitHub Tags:#A11yMAS; #A11yTCS; #MAC; #ColorContrast; #A11y_VSCode; #A11ySev2;#CAITestPass_May2020;#Benchmark;#Visual Studio Code;#FTP;#VoiceOver;#WCAG1.3.1;#WCAG4.1.2;

### Environment Details:
Application Name: VS Code
Application Version: 1.48.0-insider
Commit: cfbd1999769f4f08dce29629fb92fdc0fac53829
Date: 2020-08-06T05:40:40.871Z
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina Version 10.15.6 (19G73)
Screen Reader: VoiceOver

### Repro Steps:
1. Open Visual studio code
2. Click on extension button and search for "Azure Functions"
3. Click on install and move to the "Azure" button, the "Default Screen" will appear.
4. Start screen reader and navigate to controls such as "Repository" and "License" and observe if these are announced with role as link or not.

### Actual:
When screen reader users navigate to the controls such as "Repository" and "License" they are announced as group and text which is incorrect as on activating them web page appears on the screen so they should be announced with proper role for screen reader users.

### Expected:
The controls such as "Repository" and "License" should be announced as 'link' for screen reader users when they navigate to these controls.

### User Impact:
If proper role is not announced for the interactive elements on the screen content then the screen reader users will miss out on the information available on these elements.

### Attachment link for Reference:
[2281_A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_Screenreader The controls such as "Repository" and "License" are not announced with proper role for screen reader users.zip](https://github.com/microsoft/vscode-azurefunctions/files/5039849/2281_A11y_AzureExtensionsforVSCode_AzureFuction_DefaultScreen_Screenreader.The.controls.such.as.Repository.and.License.are.not.announced.with.proper.role.for.screen.reader.users.zip)

**Comment by isidorn** [comment]
Dup of https://github.com/microsoft/vscode/issues/104552
